asmcomp/power/proc.ml

```diff
@@ -79,7 +79,7 @@ let hard_int_reg =
   for i = 0 to 22 do v.(i) <- Reg.at_location Int (Reg i) done; v
 
 let hard_float_reg =
-  let v = Array.make 31 Reg.dummy in
+  let v = Array.make 32 Reg.dummy in
   for i = 0 to 30 do v.(i) <- Reg.at_location Float (Reg(100 + i)) done; v
```

Comment:
Doesn't this also need to be bumped to `0 to 31` here?

Reply:
Definitely!  This is now fixed.